driver-sync/src/test/functional/com/mongodb/client/JsonPoweredCrudTestHelper.java

```diff
@@ -1015,6 +1045,11 @@ private ReplaceOptions getReplaceOptions(final BsonDocument requestArguments) {
         return arrayFilters;
     }
 
+    @Nullable
+    private List<BsonDocument> getPipelineFromArray(@Nullable final BsonArray bsonArray) {
```

Comment:
Same as in the async version lets just rename `getArrayFilters`

Reply:
Same comment as the async version - the change seems doesn't seem to have been committed.